vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionAboutPageManager.java

```diff
@@ -63,4 +98,50 @@ public ExhibitionAboutPage getExhibitionAboutPage() {
         return aboutPageList != null && !aboutPageList.isEmpty() ? aboutPageList.get(0):new ExhibitionAboutPage();
     }
     
-}
+    /**
+     * This method maps the title in ExhibitionAboutPage, and add that to
+     * exhibitionTitles list for each user selected Exhibition Language.
+    */
+    public void setAboutPageTitle(LocalizedTextForm title, ExhibitionAboutPage exhibitionAboutPage) {
+        if(title!=null && !StringUtils.isEmpty(title.getText())) {
```

Comment:
I didn't get what you want me to do here.

Reply:
What I mean is that this check does not seem to make sense. If we want to avoid someone entering an empty title then this should be a check in the front end, not here (and I actually don't think we need to check that, it's fine to remove a title and have it empty to be filled later.).